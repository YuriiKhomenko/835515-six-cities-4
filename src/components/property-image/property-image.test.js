import React from "react";
import renderer from "react-test-renderer";
import PropertyImage from "../property-image/property-image.jsx";

const image = `img/apartment-01.jpg`;

it(`Render Property Image component`, () => {
  const tree = renderer
    .create(<PropertyImage
      image={image}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
