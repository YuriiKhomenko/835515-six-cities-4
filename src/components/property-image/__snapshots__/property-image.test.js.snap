// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Property Image component 1`] = `
<div
  className="property__image-wrapper"
>
  <img
    alt="img/apartment-01.jpg"
    className="property__image"
    src="img/apartment-01.jpg"
  />
</div>
`;
